modules/data.atmosphere/R/download.CRUNCEP_Global.R

```diff
@@ -79,6 +99,14 @@ download.CRUNCEP <- function(outfolder, start_date, end_date, site_id, lat.in, l
       # This throws an error if file not found
       dap <- ncdf4::nc_open(dap_file)
 
+      # confirm that timestamps match
+      stopifnot(dap$dim$time$len == ntime)
+      dap_time <- udunits2::ud.convert(dap$dim$time$vals,
+                                       dap$dim$time$units,
+                                       paste0("days since ", start_year, "-01-01"))
```

Comment:
Right? I should avoid ambiguity and add "00:00:00Z" to the output units, though.